src/as/math/MathUtils.ts

```diff
@@ -12,33 +12,41 @@
  * @see <a href="https://github.com/mrdoob/three.js/blob/master/src/math/Math">src/math/Math</a>
  */
 
-export const DEG2RAD: f32 = Mathf.PI / 180
-export const RAD2DEG: f32 = 180 / Mathf.PI
+const DEG2RAD: f32 = Mathf.PI / 180
+const RAD2DEG: f32 = 180 / Mathf.PI
 
-export function toString(source: i32, radix: i8): string {
-	let result: string = ''
+const toStringRadixSupported: bool = false // toString: The radix parameter is not supported yet.
 
-	return result
+/**
+ * Returns the respective basic value converted to a string.
+ */
+export function toString(number: i32, radix?: i8): string {
+	if (radix === null || radix > 1 && radix < 17) {
+		if (toStringRadixSupported) {
+			return number.toString(radix)
+		}
+		return _toString(number, radix)
+	}
+	throw new Error('number can be converted to string only on a radix between 2 and 36')
 }
 
-function _toHexString(integer: i32): string {
-	//number mod 16
-	//lookup mod in hex strings
-	//add to the beginning of the string
-	//divide by 16
-	//repeat while number > 0
+//digit := number % 16
+function _toString(number: i32, radix?: i8, result: string = ''): string {
+	//lookup digit character in hex string array
+	//add digit character to the beginning of the string
+	//recurse while number > 16
+	const hexValues: string[] = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F']
+	const digit: i32 = number % radix
 
-	var hexValues: string[] = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F']
-	var mod: i32 = 0
-	var result: string = ''
-
-	while (integer > 1) {
-		mod = integer % 16
-		result = hexValues[mod] + result
-		integer = integer / 16
+	if (number > radix) {
+		// convert next digit
+		_toString(number / radix, radix, hexValues[digit] + result)
 	}
+	return hexValues[digit] + result
+}
 
-	return result
+function _toHexString(number: i32, result: string = ''): string {
+	return toString(number, 16)
 }
```

Comment:
is it necessary?

Reply:
Yes, it is called here:  #85

Reply:
In generateUUID

Reply:
I see. Hmm, I guess better remove all this unnecessary helpers with unused arguments